src/Siwapp/ProductBundle/Entity/Product.php

```diff
@@ -54,6 +54,14 @@ class Product implements \JsonSerializable, Timestampable
      */
     private $items;
 
+    /**
+     * Product constructor.
+     */
+    public function __construct()
+    {
+        $this->reference = 'P'.date("Y").mt_rand(10000,99999);
```

Comment:
Also irrelevant i think? Can we please get this in a separate PR and discuss there?

Reply:
ok no problem.